### DISCUSSION TITLE
How to automatically click a specific text when text is visible?

### DISCUSSION THREAD

**Discussion by mdajnoman** [Dhruv-Techapps/acf-docs#356] [Q&A]
How to automatically click a specific text when text is visible on specific webpage?
I am trying too many things but I can't got the solution. Please help me.
Thanks in Advance.

**Comment by Dhruv-Tech-Apps** [comment]
First we will create a action to click on text on which we want to click.

[Element Finder](https://getautoclicker.com/docs/3.x/action/element-finder/) - `//*[contains(text(),"Hello")]`
[Value](https://getautoclicker.com/docs/3.x/action/value/) - ` ` blank to perform click action

Now click on [Action settings](https://getautoclicker.com/docs/3.x/action-settings/overview/)
under [retry](https://getautoclicker.com/docs/3.x/action-settings/retry/) - `-2` its for infinite try. If you want to try for limited number you can change to that specific number
under [retry Interval](https://getautoclicker.com/docs/3.x/action-settings/retry-interval/) - `5` extension will re check for that element every 5 sec. you can increase or decrease this number.

**Reply by Crywar21** [reply]
If i want to click on the first button? there are many button whit the same name or the same class, and so the automation no works well. thanks

**Reply by dharmesh-hemaram** [reply]
> If i want to click on the first button? there are many button whit the same name or the same class, and so the automation no works well. thanks

Try (//xpath)[1] for first button
